python/GafferUI/ColorChooser.py

```diff
@@ -235,7 +235,8 @@ def __componentValueChanged( self, componentWidget, reason ) :
 			a = { "r" : 0, "g" : 1, "b" : 2, "a" : 3 }[componentWidget.component]
 			newColor[a] = componentValue
 		else :
-			newColor = newColor.rgb2hsv()
+			newColor = self.__colorHsv.__class__( self.__colorHsv )
```

Comment:
The `newColor` assignment on L233 can come inside the `if` (to L235) now, making the dataflow a bit clearer.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5909/commits/b55619ce498d3c472c7791f8088de7c3fad50453